### PR TITLE
Add Excel route model conversion function

### PR DESCRIPTION
**Pull request by landont168**

### CODE REVIEW

**Review by rodrigotiscareno — CHANGES_REQUESTED** [review]
Really good stuff!! Thanks Landon. One small request, can we make this script accept two command line parameters, one the path to the raw file and two, the path to the output file? Other than that good to go

File: etl/etl_routemodel/etl_route.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Note: Need to 'pip install openpyxl'
```
Comment (probro27) [inline etl/etl_routemodel/etl_route.py:1]: You can add this to the `requirements.txt`. 
To do that, just do `pip freeze` and get all of the requirements and paste them or pipe them to `requirements.txt`
Command: `pip freeze > requirements.txt`

File: etl/etl_routemodel/etl_route.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+def save_to_csv(df, excel_file_name):
+    csv_file_name = Path(excel_file_name).with_suffix(".csv")
+    df.to_csv(csv_file_name, index=False)
+    return df
+
+
+def main():
+    # Set Excel file name
+    excel_file_name = "route-steps-2022.xlsx"
```
Comment (probro27) [inline etl/etl_routemodel/etl_route.py:31]: As @rodrigotiscareno mentioned take this as a command line input. To make it a bit better try and use the `questionary` library used by Ryan in the other files and take it using that. 
Also add the command line argument to take the output file (different from the input file)

**Review by probro27 — CHANGES_REQUESTED** [review]
@landont168 PR looks solid. I have mentioned couple of changes and also there are some linting errors. You can fix them by checking it with the **python black** formatter
Most of them can be fixed by running `black etk_route.py` (installation here: https://github.com/psf/black?tab=readme-ov-file#installation-and-usage)

**Pull request closed without merging**